UPGRADING.md

```diff
@@ -1,6 +1,300 @@
-Upgrading from previous versions
-================================
+# Migration Guide from Fast DDS v2 to Fast DDS v3
 
-If you have used *fastddsgen* to generate part of your code, it is always *recommended* to regenerate it.
+This document aims to help during the migration process from eProsima *Fast DDS version* 2 to *Fast DDS version* 3.
+For more information about all the updates, please refer to the [release notes](https://fast-dds.docs.eprosima.com/en/latest/notes/notes.html).
 
-When upgrading from a version older than 1.7.0, regeneration is **required**.
+As announced in previous minor releases, it is always *recommended* to regenerate the types with the latest *eProsima Fast DDS Gen* release.
```

Comment:
It's not always the latest! It's the latest patch associated with the corresponding Fast DDS version